pyproject.toml

```
@@ -35,6 +35,7 @@ server = [
   "aiohttp==3.11.10",
   "aiorun==2024.8.1",
   "async-timeout==5.0.1",
+  "atomicwrites==1.4.0",
```

There is 1.4.1 it seems? :thinking: 

```suggestion
  "atomicwrites==1.4.1",
```